test/parallel/test-bootstrap-modules.js

```diff
@@ -58,28 +58,32 @@ expected.beforePreExec = new Set([
   'NativeModule internal/process/promises',
   'NativeModule internal/fixed_queue',
   'NativeModule async_hooks',
-  'NativeModule internal/process/task_queues',
-  'NativeModule timers',
+  'Internal Binding blob',
+  'Internal Binding mksnapshot',
+  'Internal Binding performance',
+  'Internal Binding permission',
   'Internal Binding trace_events',
+  'Internal Binding url',
+  'NativeModule internal/console/constructor',
+  'NativeModule internal/console/global',
   'NativeModule internal/constants',
-  'NativeModule path',
+  'NativeModule internal/event_target',
+  'NativeModule internal/perf/utils',
+  'NativeModule internal/process/arch',
   'NativeModule internal/process/execution',
   'NativeModule internal/process/permission',
+  'NativeModule internal/process/platform',
+  'NativeModule internal/process/task_queues',
+  'NativeModule internal/process/version',
   'NativeModule internal/process/warning',
-  'NativeModule internal/console/constructor',
-  'NativeModule internal/console/global',
   'NativeModule internal/querystring',
-  'NativeModule querystring',
-  'Internal Binding url',
-  'Internal Binding blob',
   'NativeModule internal/url',
-  'NativeModule util',
   'NativeModule internal/webidl',
-  'Internal Binding performance',
-  'Internal Binding permission',
-  'NativeModule internal/perf/utils',
-  'NativeModule internal/event_target',
-  'Internal Binding mksnapshot',
+  'NativeModule path',
+  'NativeModule process',
```

Comment:
The cache and load time (even if it's small) would be increased by loading the added BuiltinModules. Isn't it better to keep the lookups?